### ISSUE TITLE
Hello, I am a big fan of mods that improve gameplay without changing vanila's esscence like yours, in addition to being a big fan of illagers.

### ISSUE THREAD

**Issue by Fel00** [PssbleTrngle/Brazier#31]
your mod is very good so far!! and I have some suggestions for the Crazed mob

1 - I still haven't found any of them appearing naturally in the two mansions I went to, (I don't know if it's just me or if he really isn't spawning naturally), I would suggest allowing him to be an enemy during raids but I think that would make living fire farmable which doesn't seem like a very good idea

2 - I suggest that you consider buffing Crazed or even adding new attacks like: "faster spreading green fire that dosen't destroy blocks but it causes a lot of damage", at the moment he is extremely weak and you can just avoid all damage just by walking

that's all i have to say, your mod is exactly the kind of mod i like, keep up with your great work

**Comment by PssbleTrngle** [comment]
Yes, it seems that crazed have a chance to not spawn in a mansion at all currently. I did not want to include them in raids because I wanted the give mansions something unique. 
Maybe instead of spawning them in mansions only on generation, they have a chance to spawn there later too.

I have also thought about multiple additional attacks for the crazed and I will implement something stronger, yes
